requirements.txt

```diff
@@ -57,6 +62,7 @@ smmap==0.9.0
 sockjs-tornado==1.0.1
 sourcemap==0.1.8
 tornado==2.4.1
+twilio==5.2.0
 python-twitter==1.1
 ujson==1.33
 uritemplate==0.6
```

Comment:
I checked these packages on pypi:

According to https://pypi.python.org/pypi/django-two-factor-auth:
> This project is currently unmaintained. Not recommended for production use.

django-phonenumber-field==0.7.2 doesn't mention which versions of python it supports. django-phonenumber-field==1.1.0 is python 3 compatible.

Apart from these, all other dependencies mention on pypi that they are python 3 compatible.